packages/tools/viewer-alpha/package.json

```diff
@@ -1,27 +1,35 @@
-{
-    "name": "@tools/viewer-alpha",
-    "version": "1.0.0",
-    "private": true,
-    "sideEffects": false,
-    "description": "A viewer using BabylonJS to display 3D elements natively",
-    "scripts": {
-        "start:web": "npm run serve -- --open test/apps/web/index.html",
-        "preserve": "npm run build",
-        "serve": "vite",
-        "build": "npm run compile",
-        "clean": "rimraf dist && rimraf *.tsbuildinfo",
-        "compile": "tsc -b tsconfig.build.json",
-        "test": "echo \"Error: no test specified\" && exit 1"
-    },
-    "files": [
-        "dist"
-    ],
-    "devDependencies": {
-        "@dev/core": "1.0.0",
-        "@dev/loaders": "1.0.0",
-        "chalk": "^5.3.0",
-        "source-map-loader": "^4.0.0",
-        "ts-loader": "^9.2.6",
-        "vite": "^5.3.3"
-    }
-}
+{
+    "name": "@tools/viewer-alpha",
+    "version": "1.0.0",
+    "private": true,
+    "sideEffects": false,
+    "description": "A viewer using BabylonJS to display 3D elements natively",
+    "scripts": {
+        "start:web": "npm run serve -- --open packages/tools/viewer-alpha/test/apps/web/index.html",
+        "start:bundle-test": "npm run bundle && npm run serve -- --open packages/tools/viewer-alpha/test/apps/web/bundle-test.html",
+        "preserve": "npm run build",
+        "serve": "vite",
+        "build": "npm run compile",
+        "clean": "rimraf dist && rimraf *.tsbuildinfo",
+        "compile": "tsc -b tsconfig.build.json",
+        "bundle": "rollup -c rollup.config.analyze.mjs",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "analyze": "npm run bundle && node ../../../scripts/folderSizeFlameChart dist/analyze **/*.js"
+    },
+    "files": [
+        "dist"

```

Comment:
nit (just noticed that) - an old npm issue prevents this from working correctly on linux. dist/**/*.* is the solution. It was fixed already (AFAIK), but older version of node/npm might fail (linux only...)